src/main/java/org/spongepowered/api/entity/ai/task/AITask.java

```diff
@@ -75,7 +75,7 @@
      * @param other The other task
      * @return True if it can be, false if not
      */
-    boolean canRunConcurrentWith(AITask<O> other);
+    boolean canRunConcurrentWith(AITask<? super O> other);
```

Comment:
Changed to `<?>` since you can compare `AITask<Agent>` with `AITask<Zombie>` or vice versa. There is no fixed relationship.